api/server/middleware_test.go

```diff
@@ -65,113 +58,3 @@ func TestMiddlewareChaining(t *testing.T) {
 		t.Fatal("You failed to chain correctly:", string(result))
 	}
 }
-
-func TestRootMiddleware(t *testing.T) {
```

Comment:
i think we're still using this, at least i can find it in the dataplane... need to add this one back